client/result/result.go

```diff
@@ -0,0 +1,352 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
Right yeah that's what I meant by "in the `client` package`. 

AFAIR, `result` is always used for bootstrapping, hence it made sense for it to live inside the bootstrap package. Is it also used for repair now? If so, I can see a need to move it out of bootstrap, although probably not under `client/...`